MSAL/test/integration/native_auth/requests/sign_up/MSALNativeAuthSignUpContinueIntegrationTests.swift

```diff
@@ -221,7 +221,7 @@ final class MSALNativeAuthSignUpContinueIntegrationTests: MSALNativeAuthIntegrat
 
         let response: MSALNativeAuthSignUpContinueResponse? = try await performTestSucceed()
 
-        XCTAssertNotNil(response?.signinSLT)
+        XCTAssertNotNil(response?.continuationToken)
         XCTAssertNil(response?.continuationToken)
```

Comment:
These two checks are contradictory

```
XCTAssertNotNil(response?.continuationToken)
XCTAssertNil(response?.continuationToken)
```

Reply:
Good catch, didn't run Integration tests as they are in between the two URLs but could run one of them and now I've sorted the issue. The continuation_token will come back either because it's an slt or a credential token so always not nil.